src/config/fix_inversion.json

```diff
@@ -498,6 +498,21 @@
             "invert": ".qrcode .icon-logo",
             "rules": ".qrcode { border: 8px solid black; }"
         },
```

Comment:
- Light themed *.wikia.com pages are fine, although the removal of the background image is not really necessary in my opinion.
- Dark themed *.wikia.com pages are falsely inverted, should be excluded. To take an example from the main page: http://memory-alpha.wikia.com
- wikia.com and fandom.wikia.com pages are not covered, images and dark parts are inverted.

Checkout my fix (#175)  as an alternative, which should cover (hopefully) all subpages.

Reply:
Wow, that's really good and compact version.
I'll create new request without wikia.